chat_app_be/chat_app_be/Program.cs

```diff
@@ -1,12 +1,55 @@
+using chat_app_be.Data;
+using chat_app_be.Models.Auth;
+using chat_app_be.Repositories.Interfaces;
+using chat_app_be.Services;
+using chat_app_be.Services.Interfaces;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.IdentityModel.Tokens;
+using System.Text;
+
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
 
 builder.Services.AddControllers();
+
```

Comment:
The current API has route: /api/Controller/action. (ex: /api/Auth/register)

We should lowercase the API route. We can use this option [RouteOptions.LowercaseUrls](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.routing.routeoptions.lowercaseurls?view=aspnetcore-8.0) or any other option to lower case it.